Package.swift

```diff
@@ -6,8 +6,8 @@ let package = Package(
         // PostgreSQL driver for Fluent
         .Package(url: "https://github.com/vapor/postgresql-driver.git", Version(2,0,0, prereleaseIdentifiers: ["beta"])),
```

Comment:
Go ahead and change the postgresql-driver dependency to `majorVersion: 2` as well. We can merge this PR when PostgreSQL-Driver hits 2.0.